platform/components/users/download-users.tsx

```diff
@@ -38,6 +41,12 @@ const ExportUsersButton: React.FC = () => {
           title: "Error exporting data",
           description: data.error,
         });
+      }
+      if (data.exception_empty_data) {
+        toast({
+          title: "No users to export",
+          description: "There are no users to export.",
```

Comment:
title: "No user data in this scope"
description: "Start tracking users by adding user_id when logging"
https://docs.phospho.ai/analytics/sessions-and-users